components/src/ProfileGDPR/MockHttpResponses/GET/onePolicy_notConsented_response.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_data-policies": [
+      {
+        "_element": [
+          {
+            "self": {
+              "type": "datapolicies.data-policy",
+              "uri": "/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy=",
+              "href": "http://reference.epdemos.com/cortex/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy="
+            },
+            "messages": [],
+            "links": [],
+            "_datapolicyconsentform": [
+              {
+                "self": {
+                  "type": "datapolicies.data-policy-consent-form",
+                  "uri": "/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy=/form",
+                  "href": "http://reference.epdemos.com/cortex/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy=/form"
+                },
+                "messages": [],
+                "links": [
+                  {
+                    "rel": "consentaction",
+                    "type": "datapolicies.data-policy-consent-form",
+                    "uri": "/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy=/form",
+                    "href": "http://reference.epdemos.com/cortex/datapolicies/vestri/guzdgmbrircdslkdiy4taljuie2eglkcg4ytaljrim4toqzrim2dcmrviy=/form"
+                  }
+                ],
+                "data-policy-consent": "false",
+                "policy-name": "Marketing Contact Information",
+                "policy-reference-key": "DP12346"
+              }
+            ],
+            "data-policy-consent": "false",
+            "policy-name": "Marketing Contact Information",
+            "policy-reference-key": "DP12346"
+          }
+        ]
+      }
+    ]
+  }
+  
```

Comment:
please add new line char

Reply:
Somehow git is still complaining haha
![image](https://user-images.githubusercontent.com/39800563/67422326-3ee0c080-f5a0-11e9-9996-d6b4a89d84f1.png)

Reply:
It looks like it is complaining about all the json files I added.  A quick google has turned up nothing.  Want me to keep looking, or are you okay accepting this as git error?